packages/botonic-cli/src/commands/new.ts

```diff
@@ -113,7 +113,7 @@ Creating...
     spinner.succeed()
     await this.botonicApiService.buildIfChanged(false)
     this.botonicApiService.beforeExit()
-    await exec('mv ../.botonic.json .')
+    moveSync(join('..', '.botonic.json'), join(process.cwd(), '.botonic.json'))
```

Comment:
can't we use '.' instead of '.botonic.json' for the target argument?

Reply:
i remember trying this when developing the code and wasn't working as expected.